### PR TITLE
Set product/sku images

### PR DESCRIPTION
**Pull request by shaunmaharaj**
- Set product/sku images to be fetched from S3 for catalog entry list page
- Set product/sku images to be fetched from S3 for product display page

### CODE REVIEW

File: public/modules/base/category/base.category.models.js [hunk 1]
```diff
@@ -96,10 +97,16 @@ define(function (require) {
...
-          itemObj.thumbnail = parseDefaultImg(defaultImgObj);
+          // item thumbnail by sku
+          var assetObj = {};
+          var skuName = jsonPath(itemArray[i], "$.['_code'][0]['code']")[0];
+          assetObj.absolutePath = 'https://s3-us-west-2.amazonaws.com/ep-demo-images/VESTRI_VIRTUAL/'+skuName+'.png'
+          assetObj.name = 'default-image'
+          itemObj.thumbnail = assetObj;
 
+          // // item thumbnail default (Commented out as older behaviour)
+          // var defaultImgObj = jsonPath(itemArray[i], '$._definition.._assets.._element[?(@.name="default-image")]')[0];
```
Comment (mattbishop) [inline public/modules/base/category/base.category.models.js]: Generally we don’t accept commented out code.

File: public/modules/base/item/base.item.models.js [hunk 2]
```diff
@@ -91,17 +92,26 @@ define(function (require) {
...
-          assetObj.absolutePath = defaultImage['content-location'];
-          assetObj.name = defaultImage.name;
-          assetObj.relativePath = defaultImage['relative-location'];
-          assetsListArray.push(assetObj);
-        }
+        var assetObj = {};
+       
+        // item thumbnail by sku
+        var skuName = jsonPath(item, "$.['_code'][0]['code']")[0];
+        assetObj.absolutePath = 'https://s3-us-west-2.amazonaws.com/ep-demo-images/VESTRI_VIRTUAL/'+skuName+'.png'
```
Comment (mattbishop) [inline public/modules/base/item/base.item.models.js]: Can you pick the hard coded path up as a pref or in the build?

**Comment by shaunmaharaj** [comment]
@mattbishop pushed updates based on review comments. Can you please re-review? Thank you!

**Comment by shaunmaharaj** [comment]
@mattbishop I don't have merge access to this repo. Still seeing:

```
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.
```

**Pull request merged**